tests-e2e/fixture/managedenvironment/fixture.go

```diff
@@ -91,3 +91,57 @@ func HaveCredentials(expectedEnvSpec managedgitopsv1alpha1.GitOpsDeploymentManag
 		return res
 	}, BeTrue())
 }
+
+// HaveClusterResources checks if the ClusterResources field of Environment is equal to clusterResouces.
```

Comment:
Environment => GitOpsDeploymentManagedEnvironment, I believe

Reply:
I'll make the change